azure-communication-ui/azure-communication-ui-demo-app/src/main/res/layout/activity_settings.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/settingsPage"
+    android:orientation="vertical"
+    android:layout_marginStart="@dimen/azure_communication_ui_side_margin"
+    android:layout_marginEnd="@dimen/azure_communication_ui_side_margin"
+    tools:context=".SettingsActivity">
+
+    <TextView
+        android:text="Language Settings"
+        android:textSize="24sp"
+        android:textStyle="bold"
+        android:paddingTop="@dimen/azure_communication_ui_fab_margin"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="2dp"
+        android:paddingTop="@dimen/azure_communication_ui_inner_space_padding"
+        android:background="@color/black" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        style="@style/Widget.MaterialComponents.TextInputLayout.FilledBox.Dense.ExposedDropdownMenu"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="@dimen/azure_communication_ui_inner_space_padding" >
+
+        <AutoCompleteTextView
+            android:id="@+id/autoCompleteTextView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:textSize="@dimen/azure_communication_ui_fab_margin"
+            android:paddingTop="@dimen/azure_communication_ui_fab_margin"
+            android:paddingBottom="@dimen/azure_communication_ui_fab_margin"
+            android:inputType="none"
+            android:text="English"/>
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <CheckBox
+        android:id="@+id/languageIsRTL"
+        android:text="isRTL"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:onClick="onCheckBoxTap"/>
+
+    <TextView
+        android:text="Call Settings"
+        android:textSize="24sp"
+        android:textStyle="bold"
+        android:paddingTop="@dimen/azure_communication_ui_fab_margin"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="2dp"
+        android:paddingTop="@dimen/azure_communication_ui_inner_space_padding"
+        android:background="@color/black" />
```

Comment:
Hardcoded color, use attribute from theme, probably ?attr/colorOnSurface

Otherwise this will be non-contrasted in dark mode.

Reply:
Updated the color according to theme.